pkgs/os-specific/linux/systemd/0018-timesyncd-disable-NSCD-when-DNSSEC-validation-is-dis.patch

```diff
@@ -1,4 +1,4 @@
-From 7a27556920fe1feefd17096841c8f3ca1294a1b3 Mon Sep 17 00:00:00 2001
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
```

Comment:
What's going on with this change?

Reply:
I think the instructions were not followed at some point.

We have

```
`git -c format.signoff=false format-patch v${version} --no-numbered --zero-commit --no-signature`.
```

which zeros out the commit numbers.   I guess the instructions were not followed in 255